pkg/utils/category.go

```diff
@@ -0,0 +1,51 @@
+package utils
+
+// AcceptedCategories is the list of all policy categories
```

Comment:
@gaurav-gogia , @dev-gaur , can we please update this list of categories with latest list of accepted categories?

Reply:
@kanchwala-yusuf yes, it is up to date, we have a total of 8 categories and they are all here